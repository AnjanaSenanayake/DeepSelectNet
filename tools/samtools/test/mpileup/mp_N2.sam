@SQ	SN:z	LN:13
@CO	TAGC  TTAGC  AGGT ref
@CO	TAGCAA     TTAGGT
@CO	TAGCA       TAGGT
@CO	TAGC A     T AGGT
@CO	MMMMIINNNNNIIMMMM cig
@CO	ABCDEF     GHIJKL qual
sD	0	z	1	0	4M2I5D2I4M	*	0	0	TAGCAATTAGGT	ABCDEFGHIJKL
sD	0	z	1	0	4M1I1P5D1P1I4M	*	0	0	TAGCATAGGT	ABCDEHIJKL
sD	0	z	1	0	4M1P1I5D1I1P4M	*	0	0	TAGCATAGGT	ABCDFGIJKL
sN	0	z	1	0	4M2I5N2I4M	*	0	0	TAGCAATTAGGT	ABCDEFGHIJKL
sN	0	z	1	0	4M1I1P5N1P1I4M	*	0	0	TAGCATAGGT	ABCDEHIJKL
sN	0	z	1	0	4M1P1I5N1I1P4M	*	0	0	TAGCATAGGT	ABCDFGIJKL
