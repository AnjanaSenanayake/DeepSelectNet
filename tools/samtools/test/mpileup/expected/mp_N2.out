z	1	T	6	^!.^!.^!.^!.^!.^!.	AAAAAA
z	2	A	6	......	BBBBBB
z	3	G	6	......	CCCCCC
z	4	C	6	.+2AA-5TTAGC.+2A*-5TTAGC.+2*A-5TTAGC.+2AA.+2A*.+2*A	DDDDDD
z	5	T	6	***>>>	GHGGHG
z	6	T	6	***>>>	GHGGHG
z	7	A	6	***>>>	GHGGHG
z	8	G	6	***>>>	GHGGHG
z	9	C	6	*+2TT*+2*T*+2T*>+2TT>+2*T>+2T*	GHGGHG
z	10	A	6	......	IIIIII
z	11	G	6	......	JJJJJJ
z	12	G	6	......	KKKKKK
z	13	T	6	.$.$.$.$.$.$	LLLLLL
